mqtt_exporter.py

```diff
@@ -455,10 +455,23 @@ def clear(self):
 
     def update(self, label_values, value):
         child = self.metric.labels(*label_values)
+
+        # Check if value is a string that looks like a JSON object
+        if isinstance(value, str) and value.startswith('{') and value.endswith('}'):
+            try:
+                parsed_value = json.loads(value)
```

Comment:
This will be a breaking change. I think many of the mqtt_export have some json output already, but using workarounds like I have posted in #71. 
This will break the setup for me and other users. And would need a clear announcements and Version bump and better documentation. But as written above I would prefer to see such functionality triggered by an extra config parameter.